eventconsumer/src/main/scala/com/football/notifications/events/AthenaMetrics.scala

```diff
@@ -1,189 +0,0 @@
-package com.football.notifications.events
```

Comment:
This file is replaced by `eventconsumer/src/main/scala/com/gu/notifications/events/AthenaMetrics.scala` and `commoneventconsumer/src/main/scala/com/gu/notifications/athena/Athena.scala`. I'm not sure why this had `football` in the package name 🙃